src/OrderLinesList/OrdersLinesSearchConfig.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/prefer-default-export
+export const searchableIndexes = [
+  {
+    label: 'keyword',
```

Comment:
can it be `FormattedMessage` instead of translation in component?

Reply:
Unfortunately, no. In this case, you will get [Object Object] in options